input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md

```diff
@@ -24,7 +24,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 <div class="request-for-feedback">
     <p>Specific feedback is sought on following enhancements to search parameters:
     <ul>
-        <li>Support for searching for a practitioner by reference versus chained searching using practitioner.identifier.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.</li>
+        <li>Support for searching PractitionerRole using practitioner by reference, and if a search via a chained practitioner.identifier could be a more effective approach.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-43839">FHIR-43839</a>.</li>
```

Comment:
Remove double up of "Please comment" statement.

Reply:
Done